CharlieBackend.Business/Services/SecretaryService.cs

```diff
@@ -56,8 +55,8 @@ public async Task<Result<SecretaryDto>> CreateSecretaryAsync(long accountId)
 
                     await _unitOfWork.CommitAsync();
 
-                    _bus.PubSub.Publish(new AccountApprovedEvent(account.Email,
-                                        account.FirstName, account.LastName, account.Role));
+                    await _bus.SendReceive.SendAsync("EmailRenderService", new AccountApprovedEvent(account.Email,
+                                       account.FirstName, account.LastName, account.Role));
```

Comment:
extract this into INotificationService.AccountApproved(Account account)
and remove IBus from this class